cypress/e2e/shared/dashboardsIndex.test.ts

```diff
@@ -252,6 +253,183 @@ describe('Dashboards', () => {
+          cy.getByTestID('context-menu-dashboard').click()
+        })
+
+      cy.getByTestID('context-clone-dashboard').click()
+
+      // Verify cloned dashboard contents
+
+      cy.getByTestID('page-title').should(
+        'contain.text',
+        `${localDashName} (clone 1)`
+      )
+      cy.getByTestID('variable-dropdown--Power').should('be.visible')
+      cy.getByTestID('variable-dropdown-input-typeAhead--Power').should(
+        'have.value',
+        'base'
+      )
+
+      makeGraphSnapshot().shouldBeSameAs(viewGraphOrig)
+
+      cy.getByTestID('cell--view-empty markdown').should(
+        'contain.text',
+        'The cat went here and there'
+      )
+
+      // Prep verification that changes in clone are not reflected in original,
+      // i.e. that they are fully decoupled
+      // change variable N.B. final assert is waiting on fix for #3287 see below
+      cy.getByTestID('variable-dropdown-input-typeAhead--Power').click()
+      cy.getByTestID('variable-dropdown--item')
+        .eq(4)
+        .click()
+      // const viewGraphCopy = makeGraphSnapshot()
+      cy.getByTestID('cell Name this Cell').within(() => {
+        cy.getByTestID('cell-context--toggle').click()
+      })
+
+      const replaceText =
+        '### William Blake\n#### Augeries of Innocense\n\n To see a World in a Grain of Sand\n\nAnd a Heaven in a Wild Flower\n\n...`'
+      cy.getByTestID('cell-context--note').click()
+      cy.getByTestID('markdown-editor')
+        .click()
+        .type('{ctrl}a')
+        .type(replaceText)
+
+      cy.getByTestID('save-note--button').click()
+
+      cy.getByTestID('nav-item-dashboards').click()
+
+      // Verify labels
+      cy.getByTestID('dashboard-card')
+        .eq(1)
+        .within(() => {
+          cy.getByTestID(`label--pill ${labelName}`).should('be.visible')
+          cy.getByTestID('dashboard-card--name').should(
+            'contain.text',
+            `${localDashName} (clone 1)`
+          )
+          cy.getByTestID('inline-labels--add').click()
+        })
+
+      cy.getByTestID(`label--pill ${secondLabelName}`).click()
+
+      cy.getByTestID('dashboard-card')
+        .eq(1)
+        .within(() => {
+          cy.getByTestID(`label--pill ${secondLabelName}`).should('be.visible')
+          cy.getByTestID(`label--pill--delete ${labelName}`).click()
+          cy.getByTestID(`label--pill ${labelName}`).should('not.exist')
+        })
+
+      cy.getByTestID('dashboard-card')
+        .first()
+        .within(() => {
+          cy.getByTestID(`label--pill ${secondLabelName}`).should('not.exist')
+          cy.getByTestID(`label--pill ${labelName}`).should('exist')
+        })
+
+      // Verify original is decoupled from clone
+      cy.getByTestID('dashboard-card')
+        .first()
+        .within(() => {
+          cy.getByTestID('dashboard-card--name').click()
+        })
+
+      /*
+      // TODO verify graph and variable in original once #3287 is fixed
```

Comment:
can we update this comment to include the full link to the issue? There are many repos and I know I'm going to forget which one 3287 belongs to a few weeks from now 😅

Reply:
added comment with URL to issue